documentation/src/main/asciidoc/topics/config_examples/server_network_matchaddress.xml

```diff
@@ -1,3 +1,4 @@
+<!-- Selects an IP address that matches a regular expression. -->
```

Comment:
not sure how relevant it is but  `match-address`, `match-host`, and `match-interface` work with regular expressions.
Looking at the code, the following example should work `<match-interface value="eth.*"/>`

Reply:
I was also wondering if we should put regular expressions in the examples or if it was enough to just say it in the text. probably better to go with the examples too. I'll update. thanks!

Reply:
if you want it simpler, just say in the comment it supports regular expressions. Not everyone is familiar with regex and they may get confused.

otherwise, update the examples :)